extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -1048,7 +1048,9 @@ class MultipleBackendTask extends ReplicateObject {
                     site: this.site, siteStatus: status,
                     content,
                 });
-                if (status === 'COMPLETED' && content.includes('DATA')) {
+                if ((status === 'COMPLETED' &&
+                    !sourceEntry.getReplicationIsNFS()) &&
```

Comment:
I am not fond of this solution, as this code has been introduced to fix a critical bug with transient source (https://scality.atlassian.net/browse/ZENKO-1102).

I don't recall if we decided that NFS was not compatible with transient source, if so it should not be an issue, if not we have to take this into account and see if it's acceptable.

Let's discuss this tomorrow.